### ISSUE TITLE
Rework with existing schema.org types

### ISSUE THREAD

**Issue by MikeRalphson** [webapi-discovery/rfcs#6]
See https://github.com/schemaorg/schemaorg/issues/1423#issuecomment-465274452

**Comment by nickevansuk** [comment]
Also see https://github.com/schemaorg/schemaorg/issues/1423#issuecomment-549063386 if helpful :)

**Comment by MikeRalphson** [comment]
Thanks for cross-referencing!

**Comment by nickevansuk** [comment]
@MikeRalphson do you happen to know what's going on with this? Is there an intention to progress it? What's holding this up?

**Comment by MikeRalphson** [comment]
There is an intention to progress it, just every time I get around to looking at it I run out of time.

**Comment by nickevansuk** [comment]
Is it something that someone could help with, or is this just something that needs you to find the time? Would you accept a PR, for example? Would that be useful at this stage? Or is it more design thinking that's required?

**Comment by MikeRalphson** [comment]
Any help would be very gratefully received. PRs especially welcome.

**Comment by nickevansuk** [comment]
Very happy to! Are you happy with the changes outlined in this comment https://github.com/schemaorg/schemaorg/issues/1423#issuecomment-549063386 ? If so I'll happily PR them in? Also happy to have a quick Zoom/chat or whatever if that's the quickest way to go through them and check you're happy?

**Comment by MikeRalphson** [comment]
Yes, your list in the linked issue was what I was going to try and apply, PR away and we can schedule a Zoom call if anything comes up.

**Comment by nickevansuk** [comment]
Perfect will do @MikeRalphson, thanks!

**Comment by nickevansuk** [comment]
@MikeRalphson sorry for the delay, have now [done this](https://github.com/webapi-discovery/rfcs/pull/7)! Let me know if a Zoom would be easier to talk through?

Specific point of contention is:
- `documentation` vs `endpointDescription` - should we have a separate property for machine readable documentation, or overload the existing `documentation` property?

If you're happy with the proposals, I can go ahead and PR them into https://github.com/schemaorg/schemaorg/blob/master/data/ext/pending/issue-1423.ttl and https://github.com/schemaorg/schemaorg/blob/master/data/ext/pending/issue-1423-examples.txt, which seems to be the [next step](https://github.com/schemaorg/schemaorg/issues/1423#issuecomment-465270374)?

**Comment by MikeRalphson** [comment]
I'm happy with using `documentation` for machine-readable definitions, because we have the `encodingFormat` property which will enable intelligent crawlers to know whether they can parse the target documentation before fetching it (which was my overriding concern).

**Comment by nickevansuk** [comment]
Great, thanks @MikeRalphson - in that case #7 looks like it might be there now? Let me know re: [next steps](https://github.com/schemaorg/schemaorg/issues/1423#issuecomment-465270374) above?

**Comment by MikeRalphson** [comment]
If you have time to raise those schema.org PRs, that would be great!

**Comment by MikeRalphson** [comment]
Please could you also tag me in the PR descriptions so I get subscribed?

**Comment by nickevansuk** [comment]
OK great, have done, and tagged you in also https://github.com/schemaorg/schemaorg/pull/2635, comments / review welcome!

To get the essence of the report into the PR I had to also add the following:

> The `url` property indicates a landing page for the API, which is a Web page that can be navigated to in a Web browser to gain access to the API and/or additional information.

And also use a more complex `endpointUrl` in the example.

Shall we update the report with these changes to match, or just leave them in the PR as the logical implementation of the report?

**Comment by MikeRalphson** [comment]
I'm happy to leave them just in the PR for now, we can always fold the changes back if we get feedback which indicates more work is required. Thanks again.

**Comment by nickevansuk** [comment]
No problem at all! Great to contribute. Thanks for the speedy turnaround!
